index.html

```diff
@@ -0,0 +1,1008 @@
+                </a>
+              </h4>
+              <ul>
+                <li><a href="#">The Conversation</a></li>
+                <li><span>&middot;</span></li>
+                <li><span>10 minutes ago</span></li>
+              </ul>
+            </div>
+          </article>
+          <article class="latest-news-article">
+            <a href="#" class="article-image-anchor"></a>
+            <div class="article-details">
+              <h4>
+                <a href="#">
+                  How to recover from burnout and chronic work stress, according to a psychologist
+                </a>
+              </h4>
+              <ul>
+                <li><a href="#">The Conversation</a></li>
+                <li><span>&middot;</span></li>
+                <li><span>10 minutes ago</span></li>
+              </ul>
+            </div>
+          </article>
+          <article class="latest-news-article">
+            <a href="#" class="article-image-anchor"></a>
+            <div class="article-details">
+              <h4>
+                <a href="#">
+                  How to recover from burnout and chronic work stress, according to a psychologist
+                </a>
+              </h4>
+              <ul>
+                <li><a href="#">The Conversation</a></li>
+                <li><span>&middot;</span></li>
+                <li><span>10 minutes ago</span></li>
+              </ul>
+            </div>
+          </article>
+          <article class="latest-news-article">
+            <a href="#" class="article-image-anchor"></a>
+            <div class="article-details">
+              <h4>
+                <a href="#">
+                  How to recover from burnout and chronic work stress, according to a psychologist
+                </a>
+              </h4>
+              <ul>
+                <li><a href="#">The Conversation</a></li>
+                <li><span>&middot;</span></li>
+                <li><span>10 minutes ago</span></li>
+              </ul>
+            </div>
+          </article>
+        </section>
+      </section>
+
+      <section class="funding-rounds">
+        <header class="funding-rounds-header">
+          <h2>Latest funding rounds on Index.co > </h2>
+          <a href="#">See more</a>
+          <a href="#">Get updates by email</a>
+          <a href="#">Add info</a>
+        </header>
+
+        <div class="latest-funding-view">
+          <article class="funding-1 funding">
+          
+            <a href="#" class="funding-article-title">
+              <img src="images/Albert-funding-article.jpeg" alt="Albert-funding">
+              <h4 class="fund-title">Albert</h4>
+  
+              <div>
+                <h4>$50M</h4>
+                <h5>SERIES B</h5>
+              </div>
+            </a>
+  
+            <div class="funding-label">
+              <a href="#">
+                <i class="fa fa-tag label"></i>
+                <span>Finance, +6</span>
+              </a>
```

Comment:
- [ ] Edit this link. The original page has a different color in the text you have inside the `<span>` tag.

- [ ] You could increase the line-height in this part so that the text doesn't look cramped.

Reply:
On second check, the original page had a color of color: #555 for both the label and the text which is what I already have there.